code/modules/mob/living/status_procs.dm

```diff
@@ -119,29 +442,31 @@
 		"visible_message" = message, "self_message" = self_message, "examine_message" = examine_message)
 
 /mob/living/proc/absorb_stun(amount, ignoring_flag_presence)
-	if(!amount || amount <= 0 || stat || ignoring_flag_presence || !islist(stun_absorption))
+	if(amount < 0 || stat || ignoring_flag_presence || !islist(stun_absorption))
 		return FALSE
+	if(!amount)
+		amount = 0
```

Comment:
is this even doing anything? !amount means amount == 0 anyway...

Reply:
checking for nulls probably im copying it from another effect lmao